examples/room-manager/room_service.py

```diff
@@ -0,0 +1,166 @@
+
+        self.logger.info("Room created: %s", new_room)
+
+        return new_room
+
+    def __create_peer(self, room_name: str, peer_name: str) -> PeerAccess:
+        room_id = self.room_name_to_room_id[room_name]
+
+        options = PeerOptions(
+            enable_simulcast=self.config.enable_simulcast,
+            metadata={"username": peer_name},
+        )
+        peer, token = self.fishjam_client.create_peer(room_id, options=options)
+
+        peer_access = PeerAccess(
+            room=Info(id=room_id, name=room_name),
+            peer=Info(id=peer.id, name=peer_name),
+            peer_token=token,
+            websocket_url=self.websocket_url,
+        )
+
+        self.peer_name_to_access[peer_name] = peer_access
+
+        self.logger.info("Peer created: %s", peer)
+
+        return peer_access
+
+    def __remove_room(self, room_id: str):
+        room_name = self.__find_room_name(room_id)
+        peer_accesses = self.__find_peer_access_by_room_id(room_id)
+
+        if room_name:
+            self.room_name_to_room_id.pop(room_name)
+            self.logger.info(
+                "Room deleted from cache. %s", {"name": room_name, "id": room_id}
+            )
+        else:
+            self.logger.info("Room not found in cache, id: %s", room_id)
+
+        for peer_access in peer_accesses:
+            self.peer_name_to_access.pop(peer_access.peer.name)
+
+    def __remove_peer(self, peer_id: str):
+        peer_access = self.__find_peer_access(peer_id)
+
+        if peer_access:
+            self.peer_name_to_access.pop(peer_access.peer.name)
+            self.logger.info(
+                "Peer deleted from cache. %s",
+                {"id": peer_id, "room_id": peer_access.room.id},
+            )
+        else:
+            self.logger.info("Peer not found in cache, id: %s", peer_id)
+
+    def __find_peer_access(self, peer_id) -> PeerAccess | None:
+        peer_accesses = self.peer_name_to_access.values()
+        peer_access = list(
+            filter(lambda access: access.peer.id == peer_id, peer_accesses)
+        )
+
+        return peer_access[0] if peer_access else None
+
+    def __find_peer_access_by_room_id(self, room_id: str) -> List[PeerAccess]:
+        peer_accesses = self.peer_name_to_access.values()
+        filtered_peer_accesses = list(
+            filter(lambda access: access.room.id == room_id, peer_accesses)
+        )
+
+        return filtered_peer_accesses
+
+    def __find_room_name(self, room_id: str) -> str | None:
+        room_name = None
+
+        for name, r_id in self.room_name_to_room_id.items():
+            if room_id == r_id:
+                room_name = name
+                break
+
+        return room_name
+
+    def __is_in_room(self, room: Room, peer_access: PeerAccess | None) -> bool:
+        if not peer_access:
+            return False
+
+        peers = map(lambda peer: peer.id == peer_access.peer.id, room.peers)
+        return bool(peers)
```

Comment:
This is always true if there are any peers in a room `bool([False, False]) = True`.
```suggestion
        return any(peer.id == peer_access.peer.id for peer in room.peers)
```